ansible/user-management/tasks/linux-add-user.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Checking if the user already exists in Linux
+  command: getent passwd {{ users.linux.username }}
+  register: user_check
+  ignore_errors: true
+  when: "ansible_os_family == 'RedHat' or ansible_os_family == 'Debian'"
+
+- debug:
+    var: user_check
+
+- name: Creating new user in Linux
+  user:
+    name: "{{ users.linux.username }}"
+    state: present
+    groups: "{{ users.linux.groups }}"
```

Comment:
we should check for the user and create if it doesnt already exist.

Reply:
The job will update the uid if the user already exists.